web_01.html

```diff
@@ -0,0 +1,219 @@
+            <li><a href="#">CONTACT</a></li>
+            <li><a href="#"><i class="fa-solid fa-bag-shopping"></i> $180</a>
+              <ul>
+                <li><a href="#"><img src="/images/bag.jpg" width="40px"/>2 x $20 = $40</a></li>
+                <li><a href="#"><img src="/images/woman.jpg" width="40px"/>2 x $20 = $40</a></li>
+              </ul>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="main__top">
+        <div class="main__arrow-icons">
+          <a href="#"><i class="fa-solid fa-angle-left"></i></a>
+          <a href=""><i class="fa-solid fa-angle-right"></i></a>
+        </div>
+        <div class="main__socmed-icons">
+          <ul>
+            <li><a href="https://facebook.com/" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="https://twitter.com/" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="https://google.com/" target="_blank"><i class="fa-brands fa-google-plus-g"></i></a></li>
+          </ul>
+        </div>
+        <div class="main__greetings">
+          <a href="#"><h1>Hello, my minimal friend!</h1></a>
+        </div>  
+      </section>
+      <section class="main__middle">
+        <div class="container">
+          <h1>Welcome to our official website!</h1>
+          <p>dolore alias voluptatum dolorem voluptas harum accusantium veritatis corrupti in ab fuga maiores provident deserunt sit animi totam dicta suscipit non sapiente vero voluptatibus cupiditate.</p> 
+        </div>
+      </section>
+      <section class="main__bottom">
+        <div class="container">
+          <div>
+            <i class="fa-solid fa-sliders"></i>
+            <h1>Easy Setup</h1>
+            <p>Easiest and quickest way to create a website very fast.</p>
+          </div>
+          <div>
+            <i class="fa-brands fa-html5"></i>
+            <h1>Valid HTML</h1>
+            <p>We offer W3C validated files on all of our HTML5 templates.</p>
+          </div>
+          <div>
+            <i class="fa-solid fa-chart-simple"></i>
+            <h1>Clean & Simple</h1>
+            <p>Incredibly clean design provides you powerful way to grow your business.</p>
+          </div>
+          <div>
+            <i class="fa-solid fa-mobile-screen-button"></i>
+            <h1>Responsive</h1>
+            <p>Developed to keep consistency across platforms and devices.</p>
+          </div>
+          <div>
+            <i class="fa-solid fa-heart"></i>
+            <h1>Pixel Perfect</h1>
+            <p>Based on a 12 column grid system, with every pixel in place.</p>
+          </div>
+          <div>
+            <i class="fa-solid fa-envelope-open-text"></i>
+            <h1>Fast Support</h1>
+            <p>Purchasing our templates offers you fast support and regular updates.</p>
+          </div>
+        </div>
+      </section>
+      <section class="featured__projects">
```

Comment:
`projects` or `featured-projects`. Make sure to adjust children classes as well (e.g. `project__intro` to `feature-projects`)